GREEDY_WEB/ClientApp/components/UserRegistration.tsx

```diff
@@ -5,8 +5,8 @@ import DocumentTitle from 'react-document-title';
 import { RegistrationForm } from 'react-stormpath';
 import axios from 'axios';
 import { Link, NavLink } from 'react-router-dom';
-import Cookies from 'universal-cookie';
 import FacebookLogin from 'react-facebook-login';
```

Comment:
fb log gin still not working and can confuse a user, but not a big problem now. :D